go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/prometheus-community/prom-label-proxy v0.7.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.67.1
 	github.com/prometheus-operator/prometheus-operator/pkg/client v0.67.1
-	github.com/prometheus/alertmanager v0.25.0
+	github.com/prometheus/alertmanager v0.25.1-0.20230703134743-487db1383b8c
```

Comment:
A rebase should update this line to 0.26.0

Reply:
sure for the late reply. quite busy recently:( will update the testcase this week.